unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,36 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers.js';
+
+describe('evenNumbers()', () => {
+  it('should return an array containing only even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, 10, 0]);
+  });
```

Comment:
There are a number of edge cases you could test, such as an empty array, non-integer values, etc.